pkg/providers/vsphere/vmprovider_vm_test.go

```diff
@@ -2061,6 +2061,60 @@ func vmTests() {
 			})
 		})
 	})
+
+	Context("Create/Update/Delete ISO backed VirtualMachine", func() {
+		var (
+			vm      *vmopv1.VirtualMachine
+			vmClass *vmopv1.VirtualMachineClass
+		)
+
+		BeforeEach(func() {
+			vmClass = builder.DummyVirtualMachineClassGenName()
+			vm = builder.DummyBasicVirtualMachine("test-vm", "")
+
+			// Reduce diff from old tests: by default don't create an NIC.
+			if vm.Spec.Network == nil {
+				vm.Spec.Network = &vmopv1.VirtualMachineNetworkSpec{}
+			}
+			vm.Spec.Network.Disabled = true
+		})
+
+		AfterEach(func() {
+			vmClass = nil
+			vm = nil
+		})
```

Comment:
This block seems unnecessary since these vars are initialized every time in BeforeEach.